src/SessionMonitor/index.js

```diff
@@ -37,7 +37,8 @@ export class SessionMonitor {
   }
 
   isUserOnPage(pageName) {
-    const paths = window.location.href.split('/').filter(x => x !== 'dev.html');
+    let paths = window.location.href.split('/').filter(x => x !== 'dev.html');
+    paths = paths.filter(path => path !== ""); // To account for trailing slashes
```

Comment:
Could you chain this `filter` with previous line, and also make the double quote to single quote. 
And it'd be better to add unit tests for this function :)